.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -100,95 +100,70 @@ jobs:
       fail-fast: false
       max-parallel: 16
       matrix:
-        config: ['5.4', '5.10', '6.1', '5.15', 'bpf-next']
+        name: ['1', '2', '3', '4', '5', '6', '7', '8']
```

Comment:
I just noticed the output of the tests results, can we give them a more descriptive name? https://github.com/cilium/cilium/actions/runs/11332423439

At least something similar as https://github.com/cilium/cilium/actions/runs/11332421340